packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -46,18 +54,30 @@ export const Step = ({ label, content }: TStep) => {
     );
 };
 
-export const tour_type = {
+export const tour_type: TTourType = {
     key: 'onboard_tour_',
 };
 
 export const setTourType = (param: string) => {
     tour_type.key = param;
 };
 
+export const tour_status_ended: TTourStatus = {
+    key: '',
+    toggle: '',
+    type: `${tour_type.key}status`,
+};
+
 let tour: { [key: string]: string } = {};
 let current_target: number;
```

Comment:
There is no initialisation can we put some default value?
or better to set type `number | undefined`
@farabi-deriv

Reply:
@sandeep-deriv @farabi-deriv I think it better to avoid variables without initialization because in other cases we will have warnings that we don’t need.